_posts/en/newsletters/2022-04-06-newsletter.md

```diff
@@ -0,0 +1,234 @@
+  dyncom] to the Lightning-Dev mailing list this week with a follow-up
+  to his previous proposal for [upgrading channel commitment
+  formats][topic channel commitment upgrades]---or for upgrading any
+  other setting that would affect commitment transactions.  See
+  [Newsletter #108][news108 upcom] for a summary of his previous
+  proposal.  The goal is to allow channels to remain open even when new
+  features are added to a channel, such as the ability to [use taproot
+  features][zmn post].
+
+    Osuntokun's proposal contrasts with an alternative proposal
+    described in [BOLTs #868][], implemented experimentally in
+    C-Lightning, and mentioned in [Newsletter #152][news152 cl4532].  A
+    noteworthy difference between the two proposals is that Osuntokun's
+    proposal upgrades channels even while new payments ([HTLCs][topic
+    htlc]) may be proposed, while the BOLTS #868 proposal defines a
+    quiet period during which the upgrade can occur.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME: harding to update Tuesday -->
+
+- [Bitcoin Core 23.0 RC2][] is a release candidate for the next major
+  version of this predominant full node software.  The [draft release
+  notes][bcc23 rn] list multiple improvements that advanced users and
+  system administrators are encouraged to [test][test guide] before the final release.
+
+- [LND 0.14.3-beta.rc1][] is a release candidate with several bug fixes
+  for this popular LN node software.
+
+- Rust Bitcoin 0.23.0-rc.2 FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #24118][] adds a new RPC, 'sendall', which empties a
+  wallet to the benefit of a single recipient address. The call's
+  options can be used specify additional recipients, specify a subset of
+  the wallet's UTXO pool to be used as inputs, or to maximize the
+  recipient amount by skipping dust rather than leaving no UTXOs behind.
+  'sendall' therefore offers a convenient alternative to achieve some
+  applications of 'fSubtractFeeAmount', but 'fSubtractFeeAmount' is
+  still the way to go when trying to pay a recipient who is liable for
+  transaction fees.
+
+- [Bitcoin Core #23536][] sets validation rules to always enforce Taproot
+  whenever segwit is enforced, including validation of historical blocks
+(except for block 692261 which included a transaction spending a v2 witness
```

Comment:
@glozow I'm going to s/v2/v1/ since I think this is a typo, but I couldn't quickly find the actual transaction, so please revert or let me know if you actually meant v2.

Reply:
Gah so embarrassing! 🙈 Yes, v1